src/main/java/ddingdong/ddingdongBE/domain/vodprocessing/service/FacadeVodProcessingJobServiceImpl.java

```diff
@@ -16,19 +24,42 @@ public class FacadeVodProcessingJobServiceImpl implements FacadeVodProcessingJob
 
     private final VodProcessingJobService vodProcessingJobService;
     private final FileMetaDataService fileMetaDataService;
+    private final VodProcessingNotificationService vodProcessingNotificationService;
+    private final SseConnectionService sseConnectionService;
+    private final FeedService feedService;
 
     @Override
     @Transactional
     public Long create(CreatePendingVodProcessingJobCommand command) {
         FileMetaData fileMetaData = fileMetaDataService.getById(command.fileId());
-        return vodProcessingJobService.save(command.toPendingVodProcessingJob(fileMetaData));
+        VodProcessingNotification pendingNotification =
+                vodProcessingNotificationService.save(VodProcessingNotification.pending());
+        return vodProcessingJobService.save(command.toPendingVodProcessingJob(pendingNotification, fileMetaData));
     }
 
     @Override
     @Transactional
     public void updateVodProcessingJobStatus(UpdateVodProcessingJobStatusCommand command) {
         VodProcessingJob vodProcessingJob = vodProcessingJobService.getByConvertJobId(command.convertJobId());
         vodProcessingJob.updateConvertJobStatus(command.status());
+        checkVodProcessingJobStatus(vodProcessingJob);
+    }
+
+    private void checkVodProcessingJobStatus(VodProcessingJob vodProcessingJob) {
+        ConvertJobStatus convertJobStatus = vodProcessingJob.getConvertJobStatus();
+        if (convertJobStatus != ConvertJobStatus.PENDING) {
+            checkExistingFeedAndNotify(convertJobStatus, vodProcessingJob);
+        }
```

Comment:
p3)
vodProcessingJob 내부에서 처리해도 좋을 것 같아요!
```suggestion
        if (!vodProcessingJob.isPending) {
            checkExistingFeedAndNotify(vodProcessingJob);
        }
```